src/controllers/PenaltyDetailsController.ts

```diff
@@ -24,12 +24,12 @@ const navigation = {
 };
 
 const sanitizeForm = (body: PenaltyIdentifier) => {
-    const legacyPrefixChars: string = 'PEN';
-    const penalty = body.penaltyReference;
+    const legacyPrefix: string = 'PEN';
+    const penalty = body.penaltyReference.toUpperCase();
 
     return {
         companyNumber: sanitizeCompany(body.companyNumber),
-        penaltyReference: penalty.startsWith(legacyPrefixChars) ? sanitizeLegacyPenalty(penalty) : penalty.toUpperCase()
```

Comment:
the `sanitizeLegacyPenalty` already upper cases the reference. You could have left this as it was or am I missing something?

Reply:
i think he's doing this because `startsWith()` is case sensitive